e2e/cypress/integration/bot_accounts/managing_bot_accounts_spec.js

```diff
@@ -133,7 +133,7 @@ describe('Managing bot accounts', () => {
         cy.visit('/login');
 
         // # Enter bot name in the email field
-        cy.findByPlaceholderText('Email, Username or AD/LDAP Username', {timeout: TIMEOUTS.ONE_MIN}).clear().type(botName);
+        cy.findByPlaceholderText('Email or Username', {timeout: TIMEOUTS.ONE_MIN}).clear().type(botName);
```

Comment:
1. I removed the extra cy.apiAdminLogin at L148 since it's already present in beforeEach
2. I ran the tests without license since it's not enterprise. Can you try that locally and see if you still get the LDAP issue?

Reply:
I see but I suggest to make this spec pass both for EE and TE since we can't make this TE only.

Reply:
I just moved that particular test case to enterprise without having to add additional test case in tm4j

Reply:
Thanks! Now I'm thinking about the conditional test that Rohitesh mentioned before, and something like this might make sense in doing so. But I'm good with the latest change.

Reply:
Well the thing is that case is about bot account not being able to login. The placeholder having different value for EE and TE is outside that case. There should be a new case just testing the placeholder values

Reply:
I understand, my only concern is the test should be able to run both for TE and EE. With the latest change, we'll not be able to run in TE since it's marked as `@enterprise` only and required license to test.